### DISCUSSION TITLE
如何改变Selection(选框)的定位，或者如何批量修改选框内所有节点的定位？

### DISCUSSION THREAD

**Discussion by chenlong-io** [antvis/X6#3148] [Q&A]
需要批量改变 node 节点的位置， 目前想到让需要改变位置的节点放在 Selection(选框) 里，通过改变选框的位置来批量改变选框内节点的位置。

有没有 api 可以改变选框的定位？

**Comment by yuedud** [comment]
直接getNode,然后挨个setPosition就行了吧？
